packages/schematics/angular/ai-config/index.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.dev/license
+ */
+
+import {
+  Rule,
+  apply,
+  applyTemplates,
+  chain,
+  filter,
+  mergeWith,
+  move,
+  strings,
+  url,
+} from '@angular-devkit/schematics';
+import { posix as path } from 'node:path';
+import { Schema as ConfigOptions, Tool } from './schema';
+
+const AI_TOOLS: { [key in Tool]: ContextFileInfo } = {
```

Comment:
Since now you added none, you need to exclude it.
```suggestion
const AI_TOOLS: { [key in Exclude<Tool, Tool.None>]: ContextFileInfo } = {
```